src/shared/firewall-util-nft.c

```diff
@@ -658,6 +677,10 @@ static int fw_nftables_init_family(sd_netlink *nfnl, int family) {
         if (r < 0)
                 goto out_unref;
 
+        if (family == AF_INET6) {
+                ip_type_size = sizeof(struct in6_addr);
+                ip_type = TYPE_IP6ADDR;
+        }
```

Comment:
I'd like to initialize these variables here also in the case of `family == AF_INET`:
```
if (family == AF_INET) {
    ip_type_size = ...;
    ip_type = ...;
} else {
    ip_type_size = ...;
    ip_type = ...;
}